packages/code-du-travail-api/src/server/routes/__tests__/suggest.spec.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const Koa = require("koa");
+const router = require("../suggest");
+
+const app = new Koa();
+app.use(router.routes());
+
+function getSuggestions(query) {
+  return request(app.callback()).get(`/api/v1/suggest?q=` + query);
+}
+
+function ensureSuggestionsMatchSnapshot(query) {
+  return getSuggestions(query).expect(res =>
+    expect(res.body).toMatchSnapshot()
+  );
+}
+
+test("return suggestions for re in the right format", () =>
+  getSuggestions("re")
```

Comment:
can you use async in test case ? It would be easier to read.
 ```js 

test("return suggestions for re in the right format", async () => {
  const response = await request(app.callback()).get(`/api/v1/suggest?q=re`);
  expect(response.status).toBe(200)
  expect(response.get("Content-type").toMatch(/json/);
  expect(response.body).toMatchSnapshot()
})
```

if can also create a getSuggestion helder
```js
async function getSuggestion(query) {
   return request(app.callback()).get(`/api/v1/suggest?q=${query}`);
}
```

Reply:
`getSuggestions` still return a promise, but the test cases are now using async for readability. Hope you're fine with this.